scrypto/src/types.rs

```diff
@@ -68,13 +69,14 @@ pub enum ScryptoType {
 }
 
 // Need to update `scrypto-derive/src/import.rs` after changing the table below
-const MAPPING: [(ScryptoType, u8, &str); 13] = [
+const MAPPING: [(ScryptoType, u8, &str); 14] = [
     (ScryptoType::PackageAddress, 0x80, "PackageAddress"),
     (ScryptoType::ComponentAddress, 0x81, "ComponentAddress"),
     (ScryptoType::LazyMap, 0x82, "LazyMap"),
     (ScryptoType::Hash, 0x90, "Hash"),
     (ScryptoType::EcdsaPublicKey, 0x91, "EcdsaPublicKey"),
-    (ScryptoType::EcdsaSignature, 0x92, "EcdsaSignature"),
+    (ScryptoType::EcdsaPrivateKey, 0x92, "EcdsaPrivateKey"),
```

Comment:
Can we not include private key here? If not possible, let's add a `FIXME` comment that should be removed at some point.

Reply:
Removed